test/models/operation_test.js

```diff
@@ -122,4 +122,15 @@ describe('Operation', function() {
       assertMixinSpecificationExtensionsInheritance(Operation);
     });
   });
+
+  describe('#security()', function() {
+    it('should return an array of security requirements objects', function() {
+      const d = new Operation(js);
+      expect(Array.isArray(d.security())).to.equal(true);
```

Comment:
We need to test that the array is not empty. Otherwise, in the case it is empty,  the logic inside the following `forEach` won't ever be executed.

Reply:
@smoya Thanks for the review! I have incorporated the suggestion.